src/main/java/com/kingland/bootcamp/utils/FlywayScript.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2020 Kingland Systems Corporation. All Rights Reserved.
+ */
+package com.kingland.bootcamp.utils;
+
+import org.flywaydb.core.Flyway;
+/**
+ * @author KSC
+ *
+ * Use flyway complete data migrate.
+ */
+public class FlywayScript {
```

Comment:
PR1: Please remove this main method and instead use a spring ApplicationRunner.

Reply:
@antonyhot I delete this file ,flyway in spring boot project do not need to write a  file to complete data migrate.